autosklearn/data/target_validator.py

```diff
@@ -0,0 +1,341 @@
+import typing
+import warnings
+
+import numpy as np
+
+import pandas as pd
+
+import scipy.sparse
+
+import sklearn.utils
+from sklearn import preprocessing
+from sklearn.base import BaseEstimator
+from sklearn.compose import make_column_transformer
+from sklearn.utils.multiclass import type_of_target
+
+
+class TargetValidator(BaseEstimator):
+
+    def __init__(self) -> None:
+        # If a dataframe was provided, we populate
+        # this attribute with the column types from the dataframe
+        # That is, this attribute contains whether autosklearn
+        # should treat a column as categorical or numerical
+        # During fit, if the user provided feat_types, the user
+        # constrain is honored. If not, this attribute is used.
+        self.feature_types = None  # type: typing.Optional[typing.List[str]]
```

Comment:
Is this needed for the target validator?

Reply:
Thanks for the catch, this has been removed